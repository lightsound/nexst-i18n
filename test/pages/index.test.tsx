/**
 * @jest-environment jsdom
 */
import { render } from "@testing-library/react";
import Index from "src/pages/index.page";

jest.mock("react-i18next", () => {
  return {
    useTranslation: () => {
      return {
        t: (key: any) => {
          return key;
        },
      };
    },
  };
});

describe("Index page", () => {
  it("matches snapshot", () => {
    const { asFragment } = render(<Index />, {});
    expect(asFragment()).toMatchSnapshot();
  });
});
