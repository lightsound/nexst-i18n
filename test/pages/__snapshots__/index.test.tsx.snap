// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index page matches snapshot 1`] = `
<DocumentFragment>
  <div>
    <div>
      Index
    </div>
    <button
      class="p-2"
    >
      Click me!
    </button>
  </div>
</DocumentFragment>
`;
